extension/popup/popupView.jsx

```diff
@@ -890,11 +919,36 @@ const SearchResultsContent = ({
     </div>
   );
 
+  const MusicSettingCard = () => (
+    <div value={userSettings.musicService} className="results-set">
+      {userOptions.musicServices &&
+        userOptions.musicServices
+          .filter(({ name }) => name !== "auto")
+          .map(({ name, title, imgSrc }) => (
+            <button
+              key={name}
+              value={name}
+              onChange={() => {
+                onMusicServiceChange(name);
```

Comment:
Here you pass in the name in, which would be OK too, but that doesn't match the code above, which uses `event.target.value`.

I don't think this code will work as written?